simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/BasicAuthenticator.java

```diff
@@ -0,0 +1,88 @@
+package io.prometheus.client.exporter;
+
+/**
+ * Basic HTTP authenticator
+ */
+public class BasicAuthenticator extends com.sun.net.httpserver.BasicAuthenticator {
```

Comment:
It would be good to give this class a different name so that users can't confuse `io.prometheus.client.exporter.BasicAuthenticator` and `com.sun.net.httpserver.BasicAuthenticator`.

Reply:
Thoughts on renaming to `HTTPServerBasicAuthenticator`?

Reply:
Renamed to `HTTPServerBasicAuthenticator`